### PR TITLE
Matt/route detail

### PR DESCRIPTION
**Pull request by mattbarker016**
I already merged master with @myo3 work

### CODE REVIEW

File: TCAT/Controllers/RouteDetailViewController.swift [hunk 1]
```diff
@@ -479,7 +488,7 @@ class RouteDetailViewController: UIViewController, GMSMapViewDelegate, CLLocatio
         // Place and format bottom summary label
         let summaryBottomLabel = UILabel()
         if let totalTime = Time.dateComponents(from: route.departureTime, to: route.arrivalTime).minute {
-            summaryBottomLabel.text = "Trip Duration: \(abs(totalTime)) minutes"
+            summaryBottomLabel.text = "Trip Duration: \(abs(totalTime)) minute\(totalTime == 1 ? "" : "s")"
```
Comment (AAAstorga) [inline TCAT/Controllers/RouteDetailViewController.swift:491]: thanks for determining between 1 minute and 2 minutes. I dislike when apps say 1 minutes

File: TCAT/Model/BusLocation.swift [hunk 2]
```diff
@@ -28,6 +28,8 @@ class BusLocation: NSObject, NSCoding {
     var tripID: Int = 0
     var vehicleID: Int = 0
     
+    var _iconView: UIView? = nil
```
Comment (AAAstorga) [inline TCAT/Model/BusLocation.swift]: why does this have _?

File: TCAT/Model/BusPath.swift [hunk 3]
```diff
@@ -17,56 +17,51 @@ enum PathType: String {
 
 class Path: GMSPolyline {
     
+    var waypoints: [Waypoint] = []
+    var traveledPolyline: GMSPolyline = GMSPolyline()
+    var color: UIColor = .clear
+    
+    init(waypoints: [Waypoint]) {
```
Comment (AAAstorga) [inline TCAT/Model/BusPath.swift:24]: remove empty lines

**Review by AAAstorga — APPROVED** [review]
small comments, but the rest looks good!

File: TCAT/Model/BusLocation.swift (hunk 2 above)
Comment (mattbarker016) [reply TCAT/Model/BusLocation.swift]: I don't want it to be accessed. That's the syntax for "hidden" variables. I should just make it private https://forums.developer.apple.com/thread/54256

File: TCAT/Controllers/RouteDetailViewController.swift (hunk 1 above)
Comment (mattbarker016) [reply TCAT/Controllers/RouteDetailViewController.swift:491]: 💙

**Pull request merged**